lineage/blah.py

```diff
@@ -0,0 +1,39 @@
+EL 0.006591187292408536 0.006591187292408536 1.0
+EL 0.003826507378001198 0.003826507378001198 1.0
```

Comment:
What is this?

Reply:
deleted. used it for debugging previously